### PR TITLE
Add process of expense reimbursement

### PR DESCRIPTION
**Pull request by benhylau**

### CODE REVIEW

File: finance.md [hunk 1]
```diff
@@ -104,6 +102,13 @@ You can use the [Wage Calculator](https://link.hypha.coop/wage) to determine the
 After tax withholdings and other applicable deductions (e.g. CPP & EI employee contributions), the net amount is electronically transferred to the Member's bank account on pay day using **Wagepoint**.
 Refer to the guides for [Payroll](/guides.md#payroll) to set up the payroll run for each pay period.
 
+### Expense Reimbursement
+
+Member expenses eligible for reimbursement should be submitted to the [Expense Reimbursement](https://link.hypha.coop/expenses) sheet and receipts uploaded to [Employee Expenses](https://link.hypha.coop/receipts) in the same quarter they are incurred.
```
Comment (dcwalk) [inline finance.md:107]:
```suggestion
Member expenses eligible for reimbursement should be submitted to the [Expense Reimbursement](https://link.hypha.coop/expenses) sheet and receipts uploaded to [Employee Expense Receipts](https://link.hypha.coop/receipts) in the same quarter they are incurred.
```
Name should better match link?

File: finance.md [hunk 2]
```diff
@@ -104,6 +102,13 @@ You can use the [Wage Calculator](https://link.hypha.coop/wage) to determine the
...
+Reimbursement amounts are paid out with wages on the next pay day, but does not appear on tax forms.
```
Comment (dcwalk) [inline finance.md:108]:
```suggestion
Reimbursements are paid out with wages on the next pay day, but do not appear on tax forms.
```

File: guides.md [hunk 3]
```diff
@@ -190,6 +191,43 @@ We'll now send your first email to Mailinator, a cloud inbox that's helpful for
 9. Visit [the corresponding cloud inbox](https://www.mailinator.com/v3/index.jsp?zone=public&query=hyphacoop) and confirm your message has arrived, It might take a few minutes.
 10. Don’t forget to email Infra WG as mentioned in onboarding email.
 
+## Expense Reimbursement
+
+Employees should try to submit eligible expenses in the same quarter they are incurred.
```
Comment (dcwalk) [inline guides.md:196]:
```suggestion
Employees should submit eligible expenses in the same quarter they are incurred.
```
Can use stronger language here -- 3 months is a generous guideline

File: guides.md [hunk 4]
```diff
@@ -190,6 +191,43 @@ We'll now send your first email to Mailinator, a cloud inbox that's helpful for
...
+The Finance WG reviews submitted expenses before each pay period, and pays out reimbursement amounts on the next pay day.
+
+### Submitting an expense
+
+1. Upload the expense receipt to [Employee Expenses](https://link.hypha.coop/receipts) with file name `xx-YYYY-MM-DD.ext` where `xx` is the initials of the member and `ext` the file extension, then make note of the URL.
```
Comment (dcwalk) [inline guides.md:201]: See comment above about naming and link matching

File: guides.md [hunk 5]
```diff
@@ -190,6 +191,43 @@ We'll now send your first email to Mailinator, a cloud inbox that's helpful for
...
+
+1. Open the [Expense Reimbursement](https://link.hypha.coop/expenses) sheet and fill in a new row according to instructions in the sheet.
+    - Date of Expense: `[date on your receipt]`
+    - Employee: `[employee to reimburse]`
+    - Description of Expense: `[short description of what the expense is for]`
+    - Total Amount: `[total receipt amount, including HST]`
+    - HST Amount: `[total HST on the receipt]` 
+    - Currency: `[currency of the receipt]` (default to `CAD`)
+    - Exchange Rate: `[currency exchange rate to CAD]` (default to `1.000` for CAD)
```
Comment (dcwalk) [inline guides.md:210]: How will people substantiate an exchange rate? Do they have to show that portion of CC statement?

File: guides.md (hunk 4 above)
Comment (dcwalk) [reply guides.md:201]: Also do you need CC statement? Make clear it is an itemized receipt?

File: guides.md [hunk 6]
```diff
@@ -190,6 +191,43 @@ We'll now send your first email to Mailinator, a cloud inbox that's helpful for
...
+
+1. Verify all expenses that do not have a `Pay Period of Reimbursement` selected.
+
+1. Select a `Cost Center / Project` and `Pay Period of Reimbursement`.
+
+1. Verify that the `Total Amount (CAD)` is added to the [Reimbursement Summary](http://link.hypha.coop/reimbursement) sheet to the correct `Pay Period of Reimbursement` and `Employee`.
+
+1. Verify that the amount shows up correctly on the `Expense Reimbursement` column on [Employee Payroll](http://link.hypha.coop/payroll) of the applicable pay period, so it gets entered into Wagepoint on the next payroll run.
+
+1. 🚧 Track the expense appropriately in [Quickbooks Online](https://quickbooks.intuit.com/ca/).
```
Comment (dcwalk) [inline guides.md:229]: What does the bollard mean here?

File: guides.md (hunk 4 above)
Comment (benhylau) [reply guides.md:201]: I don't think we need CC statement. Do you think we do?

File: guides.md (hunk 5 above)
Comment (benhylau) [reply guides.md:210]: I think it can't be that far off to publicly kept rates, and I'd rather not add overhead?

File: guides.md (hunk 6 above)
Comment (benhylau) [reply guides.md:229]: This part is not yet well defined :) I think after first run, when we work with bookkeeper to match accounts, we'll add this.

File: guides.md (hunk 4 above)
Comment (dcwalk) [reply guides.md:201]: I think you should get a copy for foreign currency

File: guides.md (hunk 5 above)
Comment (dcwalk) [reply guides.md:210]: The rates change from the time its posted on my CC to when my statement comes out. market rates are pretty far from bank rates

Comment (benhylau) [reply guides.md:210]: Members should put the rate they actually paid out of CC, but we can leave this to the discretion of the member. If it's not unreasonable, Finance will approve it and not ask employee for proof.

There are many scenarios where it's not a CC and the rate isn't well established. For example, when I travel I mostly pay cash, and have had to use public rates / averaged rates for claims.

```suggestion
    - Exchange Rate: `[currency exchange rate to CAD]` (default to `1.000` for CAD, you use the rate on your credit card statement or market rate on the receipt date)
```

File: guides.md (hunk 4 above)
Comment (benhylau) [reply guides.md:201]: @dcwalk is this a legal requirement or an internal process you recommend?
@YurkoWasHere your thoughts?

Comment (benhylau) [inline guides.md:201]:
```suggestion
1. Upload the expense receipt to [Employee Expense Receipts](https://link.hypha.coop/receipts) with file name `xx-YYYY-MM-DD.ext` where `xx` is the initials of the member and `ext` the file extension, then make note of the URL.
```

File: guides.md (hunk 5 above)
Comment (benhylau) [reply guides.md:210]: Discussed with @YurkoWasHere we won't enforce uploading of CC statement.

**Comment by dcwalk** [comment]
I was pointing that out for reimbursement in other currencies— Im not sure an issue for us, but in non-profits that are audited, you need to show the actual cost incurred in some way. With a forgiven currency receipt doesn’t show.

was the way we did it for Our Networks not lean enough? Literally a screen cap of a line of CC showing the conversion rate... available as soon as it posts.


Seems strange to me not to ask for evidence of exchange rate

**Comment by benhylau** [comment]
See [McGill's policy](https://www.mcgill.ca/financialservices/travel/procedures):
  >Foreign currency receipts should be denominated in the original currency, with conversion to Canadian dollars properly indicating the exchange rate used. An appropriate exchange rate may either be derived from the Bank of Canada website (based on the receipt date), or the rate charged on the credit card statement. Provide a copy of the credit card statement if the latter differs greatly from the published rates. If claiming receipts in U.S., British Pounds or Euro currency, the system will default in the exchange rate based on the receipt date entered.

My take on this is similar to [what this person says (probably not Canadian specific)](https://www.quora.com/I-m-paying-out-my-employees-expenses-that-were-in-a-different-currency-How-do-I-verify-the-conversion-that-they-paid-Should-I-request-for-their-credit-card-statement):
>An employee is not obligated to provide a credit card statement.
>
>For international purchases, my company will ask the foreign currency amount (as evidenced by the receipt) and your ultimate cost in local currency. Assuming no wild fluctuations, we take your word, subject to audit, on the conversion rate. Why keep these people on your payroll if you assume they are all dishonest?
>
>Automatically using published exchange rates to reimburse your employees would be stupid. >The purchases were made by credit card and Visa / MasterCard / AmEx do not use published exchange rates. The card associations use wholesale rates in effect usually two or three days after the purchase was actually made. Then there could be foreign transaction fees of about 3% added by some card issuers. Using a published exchange rate could only shortchange your employees and sour them against you and the company while also creating more work for you in trying to clean up the mess.
>
>If your employees always have to make reimbursable purchases, step up to the professional leagues and get a company credit card.

@dcwalk do you have a reference to this as a requirement specific to non-profits?
>in non-profits that are audited, you need to show the actual cost incurred in some way. With a forgiven currency receipt doesn’t show.

I am unaware of need to keep record of exact exchange rate. My understanding is that it's usually kept by large companies bc perhaps they can't trust all their employees?

I can ask our accountant /bookkeeper in the future, but rather keep process minimal for now.

**Pull request merged**